recipe/meta.yaml

```
@@ -13,17 +13,19 @@ source:
     - patches/drop_iyt.patch  # [python_impl == 'pypy']
 
 build:
-  number: 1
-  skip: true  # [py<310 or python_impl == 'pypy']
+  number: 2
+  python_version_independent: true  # [py>=311 and is_abi3]
   entry_points:
     - yt = yt.utilities.command_line:run_main
   script: {{ PYTHON }} -m pip install . -vv
   script_env:
     - OMP_NUM_THREADS=1
     - MAX_BUILD_CORES=2
```

You'll also need this env var to opt-into limited api builds
```suggestion
    - MAX_BUILD_CORES=2
    - YT_LIMITED_API=1
```